.travis.yml

```diff
@@ -24,5 +24,4 @@ matrix:
     - os: linux
       env: IMAGE=ubuntu:artful USE_NINJA=1
 
-script:
-  - ci/one.sh
+script: travis_retry ci/one.sh
```

Comment:
@noahdesu  - FYI, the builds sporadically fail when debian https time out or fedora mirrors timeout. this function fixes that. you might want to add it to your zlog.